rules/game_process.tex

```diff
@@ -598,3 +608,15 @@ \subsubsection{Sudden Death Shoot-Out}
   \item Otherwise, the sudden death is repeated.
   \item If after 3 sudden death penalty kicks there is still no winner, the referee will toss a coin to decide the game.
 \end{itemize}
+
+\subsection{Aborting A Penalty}
+\label{sec:aborting_penalty}
+
+If a penalty was incorrectly applied to a robot and acknowledged by the referee to be incorrect, the penalty must be aborted and a signal must be sent from the GameController, with the affected robot being unable to move, with the same process used for a Standard Removal Penalty.
+
+The robot should then be removed and placed outside the field, following the same process for the Standard Removal Penalty. Once placed, the robot should be allowed to continue play immediately after being placed.
```

Comment:
If a penalty was incorrectly applied to a robot and acknowledged by the referee to be incorrect, the robot should first be removed and placed outside the field, following the same process for the Standard Removal Penalty. Once placed, the robot should then be allowed to continue play immediately after being placed and the GameController removes its penalty.